src/PlaywrightSharp/ScriptsHelper.cs

```diff
@@ -44,7 +44,7 @@ internal static object ParseEvaluateResult(JsonElement? element, Type t)
         {
             var parseEvaluateResult = typeof(ScriptsHelper)
                 .GetMethods(BindingFlags.NonPublic | BindingFlags.Static)
-                .FirstOrDefault(m => m.Name == "ParseEvaluateResult" && m.GetGenericArguments().Any());
+                .Single(m => m.Name == "ParseEvaluateResult" && m.GetGenericArguments().Any());
```

Comment:
also: isnt isgeneric better than `GetGenericArguments().Any(`?

Reply:
Could you try implementing that?

Reply:
done

Reply:
I cached the reflected value statically